exaparser/cli.py

```diff
@@ -6,22 +6,39 @@
 
 
 @click.group()
+@click.pass_context
 @click.option('-c', '--config', help='full path to config file', type=click.Path(exists=True))
-def main(config):
+def main(ctx, config):
     if config:
         ExaParserConfig.read(config)
+    ctx.ensure_object(dict)
+    ctx.obj['handlers'] = ExaParserConfig.get("global", "data_handlers").replace(" ", "").split(",")
```

Comment:
1. I am a bit puzzled at what happened here (`ctx`, `obj['handlers']` - some explanation in a docstring could help. 
2. The `.replace(" ", "").split(",")` logic seems fragile at a first glance. Should we move to `.config`?
3. Nit: `ctx` => `click_context`, `config` -> `config_file_path`

Reply:
Provide more context to comments.